alyssa_mckee/flask_fundamentals/disapearing_ninjas/server.py

```diff
@@ -0,0 +1,22 @@
+from flask import Flask, render_template
+
+app = Flask(__name__)
+
+@app.route("/")
+def index():
+	return render_template("index.html")
+
+@app.route("/ninjas")
+def ninjas():
+	return render_template("color.html", color="tmnt.png")
+
+@app.route("/ninjas/<usercolor>")
+def color(usercolor):
+	if usercolor == "red" or usercolor== "orange"or usercolor== "blue" or usercolor== "purple":
```

Comment:
a cleaner way to right this could be: if usercolor in ["red", "orange", "blue", "purple"]